static/js/containers/ProfilePage.js

```diff
@@ -40,9 +39,7 @@ class ProfilePage extends React.Component {
       dispatch(startProfileEdit());
     }
     return dispatch(validateProfile(profile, requiredFields, messages)).then(() => {
-      dispatch(saveProfile(SETTINGS.username, profile)).then(() => {
-        dispatch(clearProfileEdit());
-      });
+      dispatch(saveProfile(SETTINGS.username, profile));
```

Comment:
you're right, the problem is that when the AJAX for the PATCH to save the profile comes back we dispatch the `clearProfileEdit` action - I'm not sure exactly what the best way forward is here, since we do want to clear the profile edit object after we save. I'll talk to george a little bit and get back to you!